### PR TITLE
Merge new code snippets from master to prod

### PR DESCRIPTION
**Pull request by siewmoi**

### CODE REVIEW

File: samples/excel/89-preview-apis/events-worksheet-activated.yaml [hunk 1]
```diff
@@ -19,7 +19,8 @@ script:
 
                 await context.sync();
 
-                OfficeHelpers.UI.notify("A handler has been registered for the OnActivate event.");   
+                OfficeHelpers.UI.notify("A handler has been registered for the OnActivate event",
+                    "Try changing a cell value in the table, and watch the console output.");
```
Comment (kbrandl) [inline samples/excel/89-preview-apis/events-worksheet-activated.yaml]: The second part of this error looks like a copy/paste error. i.e., it seems to describe the onChanged event, not the onActivate event. 

The second text string should instead say something like:   "Try selecting a different sheet, and watch the console output."

Comment (kbrandl) [reply samples/excel/89-preview-apis/events-worksheet-activated.yaml]: Also, suggest that you add this second text string the UI.notify message in the **registerOnDeactivateHandler()** function as well.

Comment (kbrandl) [reply samples/excel/89-preview-apis/events-worksheet-activated.yaml]: Finally -- if your intention in updating this UI.notify message here was to address my feedback on your previous PR ([#119](https://github.com/OfficeDev/office-js-snippets/pull/119)), that feedback was for the "Register onAdded event handler" (which is in the "Events - Worksheet changed" snippet).

"....as we discussed, whenever you have time: please update the UI notification for the "Register onAdded event handler" button click"

File: playlists/excel.yaml [hunk 2]
```diff
@@ -487,6 +487,15 @@
   group: Preview APIs
   api_set:
     ExcelApi: 1.7
+- id: excel-events-table-changed
+  name: Events - Table changed
+  fileName: events-table-changed.yaml
+  description: Add event handlers for table onDataChanged and onSelectionChanged events
```
Comment (kbrandl) [inline playlists/excel.yaml]: The format (wording) of the description that you've used for the 3 event-related snippets differ. Suggest that we use consistent wording in snippet descriptions, as seeing the inconsistent wording in the menu of samples looks odd.

This snippet (**Events - Table changed**):  
- Add event handlers for table onDataChanged and onSelectionChanged events

**Events - Worksheet changed** snippet:
- Add event handlers on the data changed and worksheet added events

**Events - worksheet activated** snippet:
- Add an event handler on the worksheet activated and deactivated events

File: samples/excel/89-preview-apis/events-worksheet-activated.yaml (hunk 1 above)
Comment (siewmoi) [reply samples/excel/89-preview-apis/events-worksheet-activated.yaml]: Kim – this is a merge from master to prod.   All the code snippets have been code reviewed.   Let’s not mix a fork review with merge from master to prod.  I can make changes next time I do a PR to merge my fork to master.

Comment (siewmoi) [reply samples/excel/89-preview-apis/events-worksheet-activated.yaml]: @kbrandl -- change is in pull 122 now

**Pull request merged**